source/cli_login.rst

```diff
@@ -9,6 +9,6 @@ After the connection between the qpc command line tool and the Quipucords server
 
     # qpc server login
 
-2. Enter the server user name and password at the prompts.  The default login is ``admin`` and the password is ``pass``.
+2. Enter the server user name and password at the prompts.  The default login is ``admin`` and the password is ``qpcpassw0rd``.
```

Comment:
change: "default login" to "default user name" 
change "and the password is" to "and the default password is"

Reply:
👍